ajax/libs/javascript.util/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "javascript.util",
+  "filename": "javascript.util.min.js",
+  "version": "0.12.12",
+  "description": "javascript.util is a port of selected parts of java.util to JavaScript which main purpose is to ease porting Java code to JavaScript.",
+  "keywords": [
+    "java",
+    "javascript",
+    "util",
+    "collection"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bjornharrtell/javascript.util.git"
+  },
+  "homepage": "https://bjornharrtell.github.io/javascript.util/doc/index.html",
```

Comment:
@hare1039 We can use the shorter url `https://bjornharrtell.github.io/javascript.util/doc`.